lib/hanami/cli/commands/app/middleware.rb

```diff
@@ -23,10 +23,16 @@ module App
         # $ bundle exec hanami middleware --with-arguments
         # /    Rack::Session::Cookie args: [{:secret=>"foo"}]
         # ```
+        #
```

Comment:
Oddly, the above code blocks render fine without redcarpet, so it's probably because of language being set.